### DISCUSSION TITLE
How we can save cost on managed k8s service

### DISCUSSION THREAD

**Discussion by satrox28** [kube-green/kube-green#283] [Q&A]
Hi,

Thanks for creating this great tool.

I would like to understand, how we can save cost on managed service like GKE, EKS and AKS as the underlying node will be running and hope they will be charging by node(cpu/memory/node disk) and not by pods.

I can understand, can save on network traffic.

**Comment by davidebianchi** [comment]
Hi @satrox28! It is possible to autoscale the nodes of the managed service. In this way, once one node is empty, it can be stopped.
[Here](https://kube-green.dev/docs/real-usecase/node-downscale/#nodes) there is an example use case, running on a managed K8s service. 
If you are interest in cluster autoscaler, [here](https://kube-green.dev/docs/FAQ/#i-have-a-couple-of-nodes-with-low-utilization-but-they-are-not-scaled-down-why) I set a couple of links.
